contracts/src/main/kotlin/com/r3/corda/lib/reissuance/contracts/ReissuanceLockContract.kt

```diff
@@ -196,7 +195,7 @@ class ReissuanceLockContract<T>: Contract where T: ContractState {
 
         requireThat {
             "Exactly one input of type ReissuanceLock is expected" using (reissuanceLockInputs.size == 1)
-            val reissuanceLockInput = reissuanceLockInputs[0].state.data as ReissuanceLock<T>
+            val reissuanceLockInput = reissuanceLockInputs[0].state.data as ReissuanceLock<*>
```

Comment:
Revert and add a @Suppress("UNCHECKED_CAST")

Reply:
Done.